.azure-pipelines/steps/install-docker-compose-v1.yml

```diff
@@ -0,0 +1,24 @@
+parameters:
+  - name: isLinux
+    type: boolean
+    default: false
+  
+  - name: 'dockerComposePath'
+    type: string
+    default: 'C:/docker-compose/docker-compose.exe'
+
+steps:
+- ${{ if eq(parameters.isLinux, true) }}:
+  - bash: |
+      sudo mkdir -p "$(dirname "${{ parameters.dockerComposePath }}")" 
+      sudo curl -SL https://github.com/docker/compose/releases/download/1.29.2/docker-compose-linux-x86_64 -o ${{ parameters.dockerComposePath }}
```

Comment:
Just noticing on this one that the release versions between linux (`v1.29.2`) differ from windows (`v1.29.1`)
Unsure if it matters, just pointing out.